average.js

```diff
@@ -1,6 +1,7 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  const nums = numbers.filter((num) => !isNaN(num));
```

Comment:
Try a more descriptive name than `nums`.
We name variables so that readers can recognize their intent without having to read the entire code.

Reply:
sure, naming could have been better!